apps/portlet-administration/src/main/webapp/groovy/webui/component/admin/UIECMAdminPortlet.gtmpl

```diff
@@ -28,10 +28,10 @@
 	    <div class="<%=(uicomponent.isShowSideBar() ? "CollapseSideBar" : "ExpandSideBar")%>" onclick="<%=uicomponent.event('ShowHide')%>"><span></span></div>
 	  </div>
 	  <div class="$sideBar">
-		  <div style="overflow: auto; width: 97%; height: 100%;">
+		  <div style="overflow: auto; width: 100%; height: 100%;">

```

Comment:
Can we take the opportunity to move this style in platform-ui ?

Reply:
yes, let me do it